### DISCUSSION TITLE
Skylander console support?

### DISCUSSION THREAD

**Discussion by golftiger13** [meltaxa/jukebox-portal#12] [Ideas]
Are you considering to support the skylander console?

**Comment by meltaxa** [comment]
An interesting idea. I can see that [capull0](https://github.com/capull0) has some repositories to interact with the Skylanders console, but via a DIY usb serial interface. A dedicated Skylanders python driver would be more ideal. Unfortunately, I cannot find one nor do I own a Skylanders to properly test.

**Reply by golftiger13** [reply]
Sounds like a challenge :-)....I will check how many Skylander portals I own... but unfortunately I am living in Europe. But I have ordered an Lego Dimension. I will see how it works and could send you the test results / data, okay ?
